contracts/proposal/dao-proposal-condorcet/src/msg.rs

```diff
@@ -0,0 +1,32 @@
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use cosmwasm_std::{CosmosMsg, Empty};
+
+use dao_macros::proposal_module_query;
+
+use crate::config::UncheckedConfig;
+
+pub type InstantiateMsg = UncheckedConfig;
+
+#[cw_serde]
+pub struct Choice {
+    pub msgs: Vec<CosmosMsg<Empty>>,
```

Comment:
what about choice title and description? 

also todo later or now: we can reconcile this type with the multiple choice option. I like "choice" as a title, simpler

Reply:
as discussed in Discord: we're going to pass on making proposals human-readable for now. we're interested in writing a secure voting system first. i've added a note to the README to this effect.